docs/website/docs/command-reference/create-namespace.md

```diff
@@ -10,10 +10,7 @@ To create a namespace you can run `odo create namespace <name>`:
 ```shell
 odo create namespace mynamespace
 ```
-
-Example - 
 ```shell
-odo create namespace mynamespace
  ✓  Namespace "mynamespace" is ready for use
  ✓  New namespace created and now using namespace: mynamespace
 ```
```

Comment:
I am not sure about this. Currently, both the section about command and output seem unrelated to each other.
I would rather keep this as the following -

-----
To create a namespace you can run `odo create namespace <name>`.
Example - 
```sh
$ odo create namespace mynamespace
 ✓  Namespace "mynamespace" is ready for use
 ✓  New namespace created and now using namespace: mynamespace
```

-----
Same comment for all the changes.

Reply:
+1. I was scrolling through `odo describe binding` and it felt weird to me. Attaching screenshot:

![image](https://user-images.githubusercontent.com/713521/181208332-75e6d1c4-d23d-43b0-9e91-c28785f7dfe1.png)

Reply:
Yup, that's intentional. But @valaparthvi has a good point and I'll update it so that it includes the commands for the examples.

The intention was to keep it separate so we don't have to add a "Example output:" each time to each command..